core/services/vrf/v2/integration_v2_test.go

```diff
@@ -477,17 +477,32 @@ func subscribeVRF(
 	consumerContract vrftesthelpers.VRFConsumerContract,
 	coordinator v22.CoordinatorV2_X,
 	backend *backends.SimulatedBackend,
-	fundingJuels *big.Int,
+	fundingAmount *big.Int,
+	nativePayment bool,
 ) (v22.Subscription, *big.Int) {
-	_, err := consumerContract.CreateSubscriptionAndFund(author, fundingJuels)
-	require.NoError(t, err)
-	backend.Commit()
+	if nativePayment {
+		_, err := consumerContract.CreateSubscriptionAndFundNative(author, fundingAmount)
+		require.NoError(t, err)
+		backend.Commit()
```

Comment:
nit/ can probably break these two lines (or at least `backend.Commit()`) out of the if/else block:

```golang
var err error
if nativePayment {
	_, err = consumerContract.CreateSubscriptionAndFundNative(author, fundingAmount)
} else {
	_, err = consumerContract.CreateSubscriptionAndFund(author, fundingAmount)
}
require.NoError(t, err)
backend.Commit()
```